code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/chartfield/ChartFieldForm.ts

```diff
@@ -698,6 +697,7 @@ export class ChartFieldForm extends Form {
       transparentCheckBox.setEnabled(scout.isOneOf(type, Chart.Type.BAR, Chart.Type.BAR_HORIZONTAL, Chart.Type.COMBO_BAR_LINE));
       accordingToValuesCheckbox.setEnabled(scout.isOneOf(type, Chart.Type.SALESFUNNEL));
       this.fulfillmentStartValuePropertyCheckbox.setEnabled(scout.isOneOf(type, Chart.Type.FULFILLMENT));
+      this.colorModeField.setPropertyDimension('enabled', 'chartType', !scout.isOneOf(type, Chart.Type.SALESFUNNEL, Chart.Type.FULFILLMENT, Chart.Type.SPEEDO, Chart.Type.VENN));
```

Comment:
The implementation is different in JS and Java (logic seems to be inverted). Is this intentional?